src/lib.rs

```diff
@@ -581,6 +591,7 @@ impl Default for BindgenOptions {
             emit_ast: false,
             emit_ir: false,
             derive_debug: true,
+            derive_default: true,
```

Comment:
Let's make it opt-in instead as discussed? Hmm, though that means that the option should be `derive-default`. I'm happy turning it on for testing so it gets more coverage (see the call to `no_unstable_rust` in `test/test.rs`).

Perhaps we should start doing something like `--derive-default={yes,no}`? Not sure how easy would that be.

What do you think?